nuxeo-core/nuxeo-core-management/src/main/java/org/nuxeo/ecm/core/management/probes/ProbeManagerImpl.java

```diff
@@ -44,10 +48,14 @@
 
     protected final Map<String, Probe> probesByShortcuts = new HashMap<String, Probe>();
 
+    protected final Map<String, ProbeInfo> probesForHealthCheck = new HashMap<String, ProbeInfo>();
```

Comment:
Please note in a Javadoc for this field that it may contain `null` values, as this is not common. It would be better to avoid doing this though.